docs/output.md

```diff
@@ -21,40 +19,60 @@ The pipeline is built using [Nextflow](https://www.nextflow.io/) and processes d
 - [Convert](#convert) - Convert reference panel to .hap and .legend files
 - [QUILT](#quilt) - Perform imputation
 - [Concatenate](#concatenate) - Concatenate all imputed chunks into a single VCF.
-- [Pipeline information](#pipeline-information) - Report metrics generated during the workflow execution
```

Comment:
We still have the pipeline and mutliqc steps.

Reply:
Yes, it is now in a different section of the `output.md`. It is in the section `##Reports`

Reply:
Ah, sorry didn't saw it ^^